### PR TITLE
Fix performance issues (complete rewrite.)

### LINKED ISSUE #1: performance issue
I think there is a performance issue with this implementation.
the code 

>`$.each(mutation.addedNodes, function(i, addedNode){
							target = target.add(addedNode);
						});`

will have very bad performance when the addedNodes length goes to thousands.
the performance might related to jquery, but we can void by just use a pure javascript array instead of the jquery object to hold the target.

You can repeat this issue at 
[https://jsfiddle.net/liuhongbo/f8moz1cz/](https://jsfiddle.net/liuhongbo/f8moz1cz/)

### PR DESCRIPTION
**Pull request by dbezborodovrp**
Resolves #1

### CODE REVIEW

**Comment by adampietrasiak** [comment]
I would merge it, but could you add initial initialization function too as 2nd argument? Sometimes it's needed to add some code that is executed only once (for example adding some DOM at the end of body/some scripts/css styles etc).

File: jquery.initialize.js [hunk 1]
```diff
@@ -1,99 +1,48 @@
-//jquery.initialize plugin by Adam Pietrasiak ( https://github.com/AdamPietrasiak/jquery.initialize )
-
+// @link https://github.com/adampietrasiak/jquery.initialize
+// @author Adam Pietrasiak
+// @author Damien Bezborodov
 ;(function($) {
-
-	$.fn.initialize = function(init, firstInit) {
```
Comment (adampietrasiak) [inline jquery.initialize.js]: In your change you didnt include `firstInit` parameter

File: jquery.initialize.js [hunk 2]
```diff
@@ -1,99 +1,48 @@
...
+        for (var j = 0; j < msobservers.length; j++) {
+            $(msobservers[j].selector).each(msobservers[j].callback);
+        }
+    });
+
+    // Observe the entire document.
+    observer.observe(document.documentElement, {childList: true, subtree: true, attributes: true});
+
+    // Handle .initialize() calls.
+    $.fn.initialize = function(callback) {
```
Comment (adampietrasiak) [inline jquery.initialize.js]: please add `firstInit` param.

**Comment by dbezborodovrp** [comment]
Thanks for doing the pull request.

Sorry, I missed the part of the API that allows `firstInit` to be passed to `.initialize`.

`firstInit` isn't really necessary, though. Wouldn't the following be equivalent:

    $("#change-class").click(function() {
        $(".wrong-class").addClass('initialize-me');
    }).each(function() {
        console.log(this);
    });

Is the same as:

    $("#change-class").click(function() {
        $(".wrong-class").addClass('initialize-me');
    }, function() {
        console.log(this);
    });

Using `.each()` seems clearer to me.

I can create a pull request that restores `firstInit` for sake of backwards compatibility though?

File: jquery.initialize.js (hunk 1 above)
Comment (dbezborodovrp) [inline jquery.initialize.js]: See #3

File: jquery.initialize.js (hunk 2 above)
Comment (dbezborodovrp) [inline jquery.initialize.js]: See #3

**Pull request merged**